src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,14 @@
+import './NavBar.css';
+import { NavLink } from 'react-router-dom';
+
+const NavBar = () => {
+  return (
+    <nav>
+      <NavLink to='/'>Choose State</NavLink>
```

Comment:
Perfect, i think these are all we need really. I also wonder if we need something more catchy than' Birds Seen'